flink-annotations/src/main/java/org/apache/flink/annotation/docs/Documentation.java

```diff
@@ -82,6 +82,7 @@
 		public static final String STATE_BACKEND_ROCKSDB = "state_backend_rocksdb";
 
 		public static final String EXPERT_CLASS_LOADING = "expert_class_loading";
+		public static final String EXPERT_DEBUGGING_AND_TUNING = "expert_debugging_and_tuning";
```

Comment:
the new section must be integrated into the documentation, by embedding the resulting .html file.